lib/features/outline/OutlineProvider.js

```diff
@@ -86,6 +86,16 @@ OutlineProvider.prototype.getOutline = function(element) {
       height: element.height + DEFAULT_OFFSET * 2
     }, OUTLINE_STYLE));
 
+  } else if (is(element, 'bpmn:EndEvent')) {
+
+    outline = svgCreate('circle');
+
+    svgAttr(outline, assign({
+      cx: element.width / 2,
+      cy: element.height / 2,
+      r: element.width / 2 + DEFAULT_OFFSET + 1
```

Comment:
Let's add a comment why `+ 1` is necessary:

> Increased stroke-width of end event grows both inward and outward which makes end event bigger than other events.

Reply:
done